doc/developer.md

```diff
@@ -0,0 +1,41 @@
+# Developer information
+
+
+## Development environment
+
+
+You can get the latest development master branch with:
+```
+    git clone https://github.com/dib-lab/sourmash.git
+```
+To install all of the necessary dependencies, do:
+```
+    pip install -r requirements.txt
+```
+Briefly, we use `py.test` for testing, and `coverage` for code
+coverage analysis.
+
+We suggest working on sourmash in a virtualenv; e.g. from within the
+sourmash clone directory, you can do:
+```
+    python -m virtualenv dev
+    . dev/bin/activate
+    make clean test
```

Comment:
You can drop all the leading spaces in each code block. In RST the indentation indicates that this is a code block. Since Markdown uses tick marks, the leading spaces aren't needed.